docker-for-mac/edge-release-notes.md

```diff
@@ -11,6 +11,20 @@ This page contains information about Docker Desktop Edge releases. Edge releases
 For information about Stable releases, see the [Stable release notes](release-notes.md). For Docker Desktop system requirements, see
 [What to know before you install](install.md#what-to-know-before-you-install).
 
+## Docker Desktop Community 2.5.4
+2020-12-07
+
+> [Download](https://desktop.docker.com/mac/edge/50534/Docker.dmg)
+
+### Upgrades
+
+- [Docker Engine 20.10.0-rc2](https://github.com/docker/docker-ce/blob/master/CHANGELOG.md#20100)
+
+### Bug fixes and minor changes
+
+- Changed the «Update and quit» menu entry to «Update and restart».
+- Fixed the manual check for updates dialog reporting a new available version number being the build number instead of the version number.
```

Comment:
Too wordy. Try:

Fixed the check for updates dialog reporting the build number instead of the version number of a new version.

Reply:
Done! 😄